routes.js

```diff
@@ -40,25 +40,27 @@ router.get('/user-type', auth.authUser, (req, res) => userTypeController.getAll(
 router.get('/user-type/:id', auth.authUser, (req, res) => userTypeController.getById(req, res));
 
 // Company routes
-router.post('/company', auth.authAdmin, (req, res) => companyController.create(req, res));
-router.get('/company', auth.authAdmin, (req, res) => companyController.getAll(req, res));
-router.get('/company/:id', auth.authAdmin, (req, res) => companyController.getById(req, res));
+router.post('/company', (req, res) => companyController.create(req, res));
```

Comment:
Why did you remove the auth middleware? I thought all of our api should be protected with authentication.

Reply:
I wrote comments in the code for as to why. At the moment, all companies are created by us. There is no use case in which an `admin`, `super-admin` level user should be able to create a new company. Every company created is assigned to a super-admin type user who then on creates admins and employees for that company. Until we agree to create a new user-type for us: ex. Super-Super-Admin I left out authentication methods.

Reply:
This also extends to that no super-admin should be allowed to delete a company.

Reply:
I realize that the way it is now, anyone can create or delete a company, but until a new user-type is agreed upon, I will leave it at this.